static/js/invite.js

```diff
@@ -270,73 +236,128 @@ function set_custom_time_inputs_visibility() {
     }
 }
 
-export function initialize() {
+function open_invite_user_modal(e) {
+    e.stopPropagation();
+    e.preventDefault();
+
+    gear_menu.close();
```

Comment:
Does this throw an exception or anything if the gear menu isn't open?

Reply:
The `close()` is called only if the gear_menu [is open](https://github.com/zulip/zulip/blob/a2ea5e859875e033dd93c88efd0c6e0f57d07735/web/src/gear_menu.js#L152-L154)